app/src/common/shared/com/igalia/wolvic/ui/widgets/WindowWidget.java

```diff
@@ -523,6 +526,22 @@ public void showPanel(@Windows.PanelType int panelType) {
         showPanel(panelType, true);
     }
 
+    public void showNewTab() {
+        if (mNewTab != null) {
+            setView(mNewTab, true);
+            mViewModel.setIsFindInPage(false);
+            final Runnable firstDrawCallback = mFirstDrawCallback;
```

Comment:
I am not sure, but I think that we need to have the same check as in `showPanel()` below.

```java
if (mRestoreFirstPaint == null && !isFirstPaintReady() && (mFirstDrawCallback != null) && (mSurface != null)) {
...
```